domains/kenquibel.json

```
@@ -3,6 +3,7 @@
     "username": "Exzort567"
   },
   "records": {
-    "CNAME": "cname.vercel-dns.com"
+    "CNAME": "cname.vercel-dns.com",
+    "TXT": "vc-domain-verify=kenquibel.is-a.dev,0a4c21768c74ace5a949"
```

please read the step 5 of this guide -> https://docs.is-a.dev/guides/vercel/
```suggestion
    "CNAME": "cname.vercel-dns.com"
```